src/app/routes/enrolment/models/enrolment-claim.ts

```diff
@@ -179,4 +184,18 @@ export class EnrolmentClaim
   private defineRequestDate(): void {
     this.requestDate = new Date((this.iclClaim as any).createdAt);
   }
+
+  private defineExpirationStatus() {
```

Comment:
I know you tested it in the app, but could you create some unit tests for those two functions?